kip-0010.md

```diff
@@ -125,57 +129,68 @@ This scenario demonstrates a script that allows for two types of spending condit
 
 1. Owner spending: The owner can spend the UTXO by providing a valid signature.
 2. Borrower spending: Anyone can spend the UTXO if they create an output with a value greater than the input by a specified threshold amount, sent to the same script.
+   How it Works and Benefits:
 
-### 2. Shared Secret Scenario
+This creates an "additive-only spending" pattern where UTXOs can only be spent by increasing their value, effectively preventing value extraction without owner authorization.
 
-This scenario demonstrates a more complex script that allows for:
+#### How it Works and Benefits
+* Uses `OpTxInputAmount` and `OpTxOutputAmount` to verify the threshold condition
+* Aligns with KIP-9's goal of controlling UTXO growth through value-based constraints
+* Enables automated services that can "borrow" funds by adding value
+* Provides spam protection through the threshold requirement while incentivizing UTXO value growth
```

Comment:
I would also add an example script (no need to use p2sh, just use the scripts directly)

Reply:
I suggest we apply the simple changes below and merge. Examples can wait for a following PR @biryukovmaxim @someone235

Reply:
@someone235 added 8efff7a4c8bd79c2e6ed99669735166625258f8f